### PR TITLE
Modify rule 4.2.12 default value

### PR DESCRIPTION
**Pull request by uri-weisman**

### CODE REVIEW

**Comment by uri-weisman** [comment]
closes https://github.com/elastic/csp-security-policies/pull/88

**Review by CohenIdo — APPROVED** [review]

File: compliance/cis_k8s/rules/cis_4_2_12/data.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 metadata:
   id: 46d69440-3946-50f6-83b3-a0987551afa2
-  name: Verify that the RotateKubeletServerCertificate argument is set to true (Manual)
+  name: Verify that the RotateKubeletServerCertificate argument is set to true or it's not present (Manual)
```
Comment (eyalkraft) [inline compliance/cis_k8s/rules/cis_4_2_12/data.yaml:3]: please don't change that as this is thew official name of the rule by CIS and not our own name

File: compliance/cis_k8s/rules/cis_4_2_12/data.yaml [hunk 2]
```diff
@@ -38,7 +38,7 @@ metadata:
   impact: |
     None
   default_value: |
-    By default, kubelet server certificate rotation is disabled.
+    By default, kubelet server certificate rotation is enabled.
```
Comment (eyalkraft) [inline compliance/cis_k8s/rules/cis_4_2_12/data.yaml:41]: please open a PR to update that here: https://github.com/elastic/integrations/blob/8dd73d610ab2d2ffdd3df90b1b2fafd14ca37b7d/packages/cloud_security_posture/kibana/csp_rule_template/46d69440-3946-50f6-83b3-a0987551afa2.json#L13

**Review by eyalkraft — CHANGES_REQUESTED** [review]
Nice! small requests

File: compliance/cis_k8s/rules/cis_4_2_12/data.yaml (hunk 2 above)
Comment (uri-weisman) [reply compliance/cis_k8s/rules/cis_4_2_12/data.yaml:41]: On it

**Review by eyalkraft — APPROVED** [review]

**Pull request merged**